### PR TITLE
add vol mount

### PR DESCRIPTION
**Pull request by Tianhao-Gu**

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -63,4 +63,6 @@ services:
     environment:
       - NOTEBOOK_PORT=4041
       - SPARK_MASTER_URL=spark://spark-master:7077
-      - SPARK_DRIVER_HOST=spark-notebook
\ No newline at end of file
+      - SPARK_DRIVER_HOST=spark-notebook
+    volumes:
+      - ./cdm_shared_workspace:/cdm_shared_workspace
```
Comment (MrCreosote) [inline docker-compose.yaml:68]: Where are you mounting from in rancher?

File: docker-compose.yaml
Comment (MrCreosote) [inline docker-compose.yaml:1]: At some point we'll probably have to think about how different users would interact with this but that's much later

File: docker-compose.yaml (hunk 1 above)
Comment (Tianhao-Gu) [reply docker-compose.yaml:68]: in Rancher I just do `/cdm_shared_workspace:/cdm_shared_workspace`.

Comment (MrCreosote) [reply docker-compose.yaml:68]: Where does the data live though? In the VM the notebook is running on?

Comment (Tianhao-Gu) [reply docker-compose.yaml:68]: I think so. Double confirming with devops team in slack.

**Comment by MrCreosote** [comment]
is this still relevant?

**Comment by Tianhao-Gu** [comment]
> is this still relevant?

no. Just closed.

**Pull request closed without merging**